### PR TITLE
Booting coreboot on Intel Comet Lake S RVP8

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+development. Typically there are a lot of connectors on those boards, that let
+the users attach various peripherals and implement applications for their
+particular needs.
+
+![Confidential](/img/confidential.png)
+
+The disadvantage of RVPs is that they are not available in public. You need to
+have a CNDA signed with Intel to obtain the platform itself as well as its
+schematics and design files. So you probably will not be able to reproduce my
+results. Anyway I think it will help you with booting a modern platform using
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md]: Added your suggestion. Anything else I could add?

File: blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+is corrupted, improper jumper settings or the worst scenario, platform is
+damaged. But in my situation, nothing like this happened (typically Intel
+refurbished the platform before shipping by flashing the reference firmware,
+setting jumpers to default positions, etc.). Now next step would be to backup
+the golden firmware image, that is proven to work, in case we will brick the
+platform and need recovery. You may use the flashrom and internal programmer
+and read the binary back (but not recommended) or use external programmer to
+dump the contents of the SPI flash. I personally recommend the latter, because:
+
+- flashrom may not yest support the chipset
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md]: Done

File: blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+No microcode... Microcode files are automatically added to coreboot image for
+given microarchitecture from intel-microcode submodule. If the microcode
+updates were not included that means, there is probably no microcode in the
+repository yet... Trying to update to recent revision did not help either. What
+now? Without microcode, we will not boot probably either. In such case our
+golden firmware image and UEFITool will come with help. UEFI Tool is a cool
+UEFI image explorer which allows extraction of the UEFI modules. That also
+applies to raw files and microcode. So let's open the binary in the tool and
+search for microcode!
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md]: @KBanecka cc

File: blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+
+## Summary
+
+The road of a firmware developer is not easy at the beginning. But with time
+you earn experience and a sort of intuition that helps you locate and resolve
+the problems pretty quickly. The more problems you resolve the more knowledge
+you gather and things become more and more easier. Unfortunately booting newer
+microarchitectures will not become simpler, so it is good to experiment with
+new hardware, so that when real products and project come, you know what
+problems to expect and how to deal with them.
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply blog/content/post/2020-08-21-booting-coreboot-on-cometlake.md:299]: Like this?

**Review by pietrushnic — APPROVED** [review]

**Review by KBanecka — APPROVED** [review]

**Pull request merged**